### PR TITLE
Misc changes for desktop GL.

### PR DESCRIPTION
**Pull request by jacereda**

### CODE REVIEW

File: transitions/pixelize.glsl [hunk 1]
```diff
@@ -2,8 +2,8 @@
 // License: MIT
 // forked from https://gist.github.com/benraziel/c528607361d90a072e98
 
-uniform ivec2 squaresMin/* = ivec2(20) */; // minimum number of squares (when the effect is at its higher level)
-uniform int steps /* = 50 */; // zero disable the stepping
```
Comment (gltransitions) [inline transitions/pixelize.glsl]: uniform 'squaresMin' default value '= ivec2(20) // minimum number of squares (when the effect is at its higher level)' does not parse as GLSL code: unexpected EOF at line 1

Comment (gltransitions) [inline transitions/pixelize.glsl]: uniform 'squaresMin' has not declared any commented default value.
Example: uniform ivec2 squaresMin; // = ivec2(1, 2);

File: transitions/pixelize.glsl [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
+uniform ivec2 squaresMin; // = ivec2(20) // minimum number of squares (when the effect is at its higher level)
```
Comment (gltransitions) [inline transitions/pixelize.glsl]: uniform 'steps' default value '= 50 // zero disable the stepping' does not parse as GLSL code: unexpected EOF at line 1

Comment (gltransitions) [inline transitions/pixelize.glsl]: uniform 'steps' has not declared any commented default value.
Example: uniform int steps; // = 42;

**Comment by gre** [comment]
@jacereda can you move the comment in a preceding line? Our parser does not support the comment inside the default value (which could also probably be fixed)

**Comment by jacereda** [comment]
Sure, moved the comments.

**Pull request merged**